src/replayToMovie.js

```diff
@@ -210,7 +215,7 @@ module.exports.renderImages = async (replay, writer) => {
         lastBackground = frame.bg;
         effect.draw(frame.context);
       } catch (err) {
-        // pass
+        renderSegment.addError(err);
```

Comment:
Why do we swallow exceptions here? Is the intention to render what we can in the video, even if it may not be fully correct, rather than failing and producing no video?

Reply:
precisely! In this case, we would end up rendering a frame with the sprites (which are known to be stable) but without the background effect (which are significantly less stable)